rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/domain/model/objectstore/TestBase.java

```diff
@@ -64,7 +74,7 @@ public void before() {
         client = ClientBuilder.newClient();
 
         server = new UndertowJaxrsServer().start();
-        server.deploy(Coordinator.class);
+        server.deploy(LRACoordinator.class);
```

Comment:
Same comment here as per LRATest "Please can you explain why is it deploying this test LRACoordinator and not the new JaxRsApplication?"